src/Frontend/Themes/Fork/Core/Layout/Templates/Base.html.twig

```diff
@@ -0,0 +1,72 @@
+{% include "Core/Layout/Templates/Head.html.twig" %}
+
+<body class="{{ LANGUAGE }}" itemscope itemtype="http://schema.org/WebPage">
+<div id="root">
+  {% block cookies %}
+    {% include "Core/Layout/Templates/Cookies.html.twig" %}
+  {% endblock %}
+
+  <div class="header header-blue" style="background-image: url('{{ FRONTEND_FILES_URL }}/pages/images/source/{{ page.data.image }}');">
+    {% block navigation %}
+      <nav class="navbar navbar-default navbar-static-top">
+        <div class="container">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-1" aria-expanded="false" aria-controls="navbar">
+              <span class="sr-only">{$lblToggleNavigation}</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="/{{ LANGUAGE }}">{{ siteTitle }}</a>
+          </div>
+          <div id="navbar-1" class="navbar-collapse collapse">
```

Comment:
Use a more descriptive id like `navbar-main`, because this doesn't say much and even suggests navbar-2, navbar-3, navbar-4 (and so on) also exist.